From 66c45fa66742cb4aa01078a448312bb4679db4f0 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 3 Feb 2016 17:21:11 +0500
Subject: [PATCH] init-install.sh: correctly handle mmc device check

The cleanup in 68d8f6d2 did not update the mmc device
checking code to generate prefix and other required
bits which makes the installation to fail as it
does not generate correct partition names.
The check is now updated to handle that accordingly.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 init-install.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git init-install.sh init-install.sh
index 72c4fd8..1856663 100644
--- init-install.sh
+++ init-install.sh
@@ -151,7 +151,7 @@ swap_start=$((rootfs_end))
 # 2) they are detected asynchronously (need rootwait)
 rootwait=""
 part_prefix=""
-if [ ! "${device#mmcblk}" = "${device}" ]; then
+if [ ! "${device#\/dev\/mmcblk}" = "${device}" ]; then
     part_prefix="p"
     rootwait="rootwait"
 fi
-- 
1.9.1

